Litmus-Leader.md

```diff
@@ -0,0 +1,29 @@
+# Litmus Leader Program
+The Litmus Leader Program aims to identify community contributors & leaders in the Litmus community and reward them for their valuable contribution to Cloud-Native world by contributing to the LitmusChaos project.
+This program is the a Rewards and Recognitions campaign for the LitmusChaos project to award our most prominent contributors in various fields like development, advocacy, writing, content creation, etc. and be an example for others.
+
+----------------------
+
+## Leadway to Litmus Leader 💥:
+
+1. Active Participation: We encourage you to be participative in feedback and attending our Monthly Community Meetups, GitHub LitmusChaos discussions, and on other handles as well (Twitter, LinkedIn, Slack, etc.)
+
+2. Active Advocacy: Talk/blog about your work/contributions/experience at LitmusChaos openly on your social media handles. Bring in referrals to expand our network.
```

Comment:
```suggestion
2. Active Advocacy: Talk/blog about your work/contributions/experience at LitmusChaos openly on your social media handles. 
```

Advocacy is expected to organically cause adoption. Not sure if folks need to bring in referrals?